WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -4,48 +4,53 @@
   <% if !@competition.registration_opened? %>
     <% if @competition.registration_not_yet_opened? %>
       <div class="alert alert-info">
-        Registration will open in <strong><%= distance_of_time_in_words_to_now(@competition.registration_open) %></strong> on <%= wca_local_time(@competition.registration_open) %>.
+        <% #This internationally works because datetime.distance_in_words.x_days is translated.%>
```

Comment:
Indeed, the `distance_of_time_in_words_to_now` will look for the localized version of the key `datetime.distance_in_words.x_days`; [here](https://github.com/viroulep/worldcubeassociation.org/blob/i18n/WcaOnRails/config/locales/rails.fr.yml#L84) for French, and in a dictionary loaded by default for English.